### PR TITLE
new containment blog post

### PR DESCRIPTION
**Pull request by estelle**
new blog post for the containment pages I did.

### CODE REVIEW

File: pages/content/posts/css-containment.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+We created a new [CSS containment module](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Containment) landing page. The module landing page includes a reference with links to all the properties, at-rules and descriptors, CSS functions, events, interfaces defined in the CSS containment specification, and guides. All of these references were reviewed and updated as necessary.
+
+The larger component of this project included updating the existing containment guides and creating new ones.  We also created a new guide.
+
+### CSS containment guides
+
+There are now three CSS containment guides on MDN:
+
+- : [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries)
```
Comment (captainbrosset) [inline pages/content/posts/css-containment.md:30]: Definition lists don't seem to work correctly on the blog.

image

Maybe use a UL instead?

Comment (estelle) [reply pages/content/posts/css-containment.md]: thanks. Went with UL.

**Review by captainbrosset — APPROVED** [review]
Great article. Feel free to use or ignore these suggestions.

File: pages/content/posts/css-containment.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+The larger component of this project included updating the existing containment guides and creating new ones.  We also created a new guide.
+
+### CSS containment guides
+
+There are now three CSS containment guides on MDN:
+
+* [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries)
+
+  We updated this introductory `@container` queries guide, which discusses size queries, naming queries, and container query length units, which are well supported. We left in the fallback for now, but look forward to removing it!
```
Comment (estelle) [inline pages/content/posts/css-containment.md:32]:
```suggestion
  We updated this introductory `@container` queries guide, which discusses size queries, naming queries, and container query length units, which are well supported. We left in the fallbacks for non-supporting browsers for now, and look forward to removing it soon when it's fully supported!
```

File: pages/content/posts/css-containment.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## CSS containment project
+
+The first sub-project completed was [CSS containment](https://github.com/openwebdocs/project/issues/195). CSS containment is part of both [Interop 2022](https://wpt.fyi/interop-2022) and [Interop 2023](https://wpt.fyi/interop-2023), and is still being worked on as part of [Interop 2024](https://wpt.fyi/interop-2024).
+
+Containment improves rendering performance by isolating sections of content for delayed layout and rendering. Container queries, also defined in the CSS containment specification, enable targeting styles based on a container's size or style features.
+
+We created a new [CSS containment module](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Containment) landing page. The module landing page includes a reference with links to all the properties, at-rules and descriptors, CSS functions, events, interfaces defined in the CSS containment specification, and guides. All of these references were reviewed and updated as necessary.
```
Comment (estelle) [inline pages/content/posts/css-containment.md:22]:
```suggestion
We created a new [CSS containment module](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Containment) landing page and guides. The module landing page includes a reference with links to all the properties, at-rules and descriptors, CSS functions, events, interfaces defined in the CSS containment specification, and guides. All these references were reviewed and updated as necessary.
```

File: pages/content/posts/css-containment.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## CSS containment project
+
+The first sub-project completed was [CSS containment](https://github.com/openwebdocs/project/issues/195). CSS containment is part of both [Interop 2022](https://wpt.fyi/interop-2022) and [Interop 2023](https://wpt.fyi/interop-2023), and is still being worked on as part of [Interop 2024](https://wpt.fyi/interop-2024).
+
+Containment improves rendering performance by isolating sections of content for delayed layout and rendering. Container queries, also defined in the CSS containment specification, enable targeting styles based on a container's size or style features.
+
+We created a new [CSS containment module](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Containment) landing page and guides. The module landing page includes a reference with links to all the properties, at-rules and descriptors, CSS functions, events, interfaces defined in the CSS containment specification, and guides. All these references were reviewed and updated as necessary.
+
+The larger component of this project included updating the existing containment guides and creating new ones.  We also created a new guide.
```
Comment (estelle) [inline pages/content/posts/css-containment.md:24]:
```suggestion
The larger component of this project included updating the existing containment guides and creating new ones.
```

File: pages/content/posts/css-containment.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+### CSS containment guides
+
+There are now three CSS containment guides on MDN:
+
+* [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries)
+
+  We updated this introductory `@container` queries guide, which discusses size queries, naming queries, and container query length units, which are well supported. We left in the fallback for now, but look forward to removing it!
```
[the remark this answers is not in the document]
Comment (estelle) [reply pages/content/posts/css-containment.md]: i just meant the fallback section. That wasn't clear, so updated this differently. thank you for catching that

File: pages/content/posts/css-containment.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
+---
+author: "Estelle Weyl"
+title: "CSS containment"
+date: "2024-04-09"
```
Comment (Elchi3) [inline pages/content/posts/css-containment.md:4]:
```suggestion
date: "2024-04-10"
```

**Review by Elchi3 — APPROVED** [review]
Great post, should we publish today?

**Pull request merged**